assets/js/base/components/cart-checkout/shipping-rates-control/index.tsx

```diff
@@ -172,14 +173,26 @@ const ShippingRatesControl = ( {
 			) }
 			showSpinner={ true }
 		>
-			<ExperimentalOrderShippingPackages.Slot { ...slotFillProps } />
-			<ExperimentalOrderShippingPackages>
+			{ isEditor ? (
 				<Packages
 					packages={ shippingRates }
 					noResultsMessage={ noResultsMessage }
 					renderOption={ renderOption }
 				/>
-			</ExperimentalOrderShippingPackages>
+			) : (
+				<>
+					<ExperimentalOrderShippingPackages.Slot
+						{ ...slotFillProps }
+					/>
```

Comment:
It would yes, but there is a known conflict/issue @senadir has been wrestling with within the editor context which this works around. Slots are either non-functional or cause duplication of other components. This does not impact frontend.

Reply:
This is going to come up again with #4447 so let's tackle it then.